apps/web/src/pages/content-only/[...slug].tsx

```diff
@@ -52,7 +52,10 @@ export default renderedPageNoHooks<SlugProps>(({ pageData }) => {
       serloEntityData={{ entityId }}
       authorization={pageData.authorization}
     >
-      <LazyIframeResizer />
+      <Script
+        src="https://cdn.jsdelivr.net/npm/@open-iframe-resizer/core@1.2.1/dist/index.min.js"
```

Comment:
could we host this on serlo.org to avoid any privacy concerns?

Reply:
I think so. Just have not done it yet.